### PR TITLE
Support passing in information about external functions in lg:analyze

### PR DESCRIPTION
**Pull request by cosmicshuai**
closes: #1220 
With --external-functions flag, user can pass in external functions to lg:analyze command. This will help users avoid `has no evaluator` error

### CODE REVIEW

File: packages/lg/src/commands/lg/analyze.ts [hunk 1]
```diff
@@ -37,6 +39,13 @@ export default class AnalyzeCommand extends Command {
     const lgFilePaths = Helper.findLGFiles(flags.in, flags.recurse)
 
     Helper.checkInputAndOutput(lgFilePaths)
+    if (flags['external-functions']) {
+      const functionList = flags['external-functions'].split(',')
```
Comment (joshgummersall) [inline packages/lg/src/commands/lg/analyze.ts:43]: Does this work for "function1,function2" as well as "function1 , function2"?

File: packages/lg/test/commands/lg/analyze.test.ts [hunk 2]
```diff
@@ -16,9 +16,9 @@ const  verifiedFolderPath = './../../fixtures/verified'
 const  generatedFolder = path.join(__dirname, generatedFolderPath)
 
 describe('lg:analyze lg file', async () => {
-  after(async function () {
-    await fs.remove(generatedFolder)
-  })
+  // after(async function () {
```
Comment (joshgummersall) [inline packages/lg/test/commands/lg/analyze.test.ts:19]: Why are these commented out?

**Comment by munozemilio** [comment]
@scheyal could you please approve from PM side?

File: packages/lg/test/commands/lg/analyze.test.ts (hunk 2 above)
Comment (cosmicshuai) [reply packages/lg/test/commands/lg/analyze.test.ts]: By mistake. I've undo it

File: packages/lg/src/commands/lg/analyze.ts (hunk 1 above)
Comment (cosmicshuai) [reply packages/lg/src/commands/lg/analyze.ts:43]: I don't think so, "function1 , function2", the cli will take , function2 as another argument

**Review by chrimc62 — APPROVED** [review]
:shipit:

**Review by joshgummersall — APPROVED** [review]

**Review by munozemilio — APPROVED** [review]

**Pull request merged**